### ISSUE TITLE
Allow node selector to be copied

### ISSUE THREAD

**Issue by dufferzafar** [titusjan/astviewer#2]
Each node should have a right click context menu item "Copy node selector" that will copy python code like `node.body[0].value.s` etc.

This is similar to what the web browser console offers for DOM - copying of CSS selectors.

![node selector](https://user-images.githubusercontent.com/1449512/29994021-76c1864a-8fb5-11e7-9b73-ed588f4b9559.png)

**Comment by titusjan** [comment]
Good idea. I'll keep in mind and will probably implement it when I make another release. However, for the moment I am giving priority to other projects, so I this won't be done any time soon. Sorry.

**Comment by titusjan** [comment]
Closed in v1.1.4
